hoot-core/src/main/cpp/hoot/core/io/ApiDbReader.cpp

```diff
@@ -110,11 +171,8 @@ void ApiDbReader::_addTagsToElement(shared_ptr<Element> element)
         double tv = tags.getLength("accuracy").value();
         element->setCircularError(tv);
         ok = true;
-        LOG_TRACE(
-          "Set circular error from accuracy tag to " << tv << " for element with ID: " <<
-          element->getId());
       }
-      catch (const HootException& e)
+      catch (const HootException& /*e*/)
```

Comment:
Not being a c++ guy this may be fine, but it caught my eye.  What's the affect of commenting out the variable name in the catch?